back-end/src/sciler/handler/handler.go

```diff
@@ -13,10 +13,10 @@ import (
 
 // Message is a type that follows the structure all messages have, described in resources/message_manual.md
 type Message struct {
-	DeviceID string                 `json:"device_id"`
-	TimeSent string                 `json:"time_sent"`
-	Type     string                 `json:"type"`
-	Contents map[string]interface{} `json:"contents"`
+	DeviceID string      `json:"device_id"`
+	TimeSent string      `json:"time_sent"`
+	Type     string      `json:"type"`
+	Contents interface{} `json:"contents"`
```

Comment:
this is still a map, at least in message manual

Reply:
Then I shall change the message manual

Reply:
message manual says doesn't say this has to be a map, in case of type instruction it is a list